robot/Cumulus/tests/draft/tdtm_username_exclude.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Update a Trigger Handler to Exclude a Username
+    [tags]  unstable
+
+    # Create a Contact so Trigger Handler records are created
+    &{account} =                 API Create Organization Account
+
+    # Navigate to Recurring Donation Trigger Handler Record and exclude Scratch User
+    @{triggerhandler} =          Salesforce Query             Trigger_Handler__c
+    ...                          select=Id
+    ...                          Class__c=RD_RecurringDonations_TDTM
+
+    @{scratchuser} =             Salesforce Query             User
+    ...                          select=Username
+    ...                          Name=User User
+
+    Go To Record Home            ${triggerhandler}[0][Id]
```

Comment:
Use API instead of UI to update Trigger Handler record based on @skristem 's suggestion.